### ISSUE TITLE
unable to install MariaDB

### ISSUE THREAD

**Issue by ashish-gupta-2** [mariadb-operator/mariadb-operator#327]
**Describe the bug**

while running this command 
```
kubectl apply -f examples/manifests/mariadb_v1alpha1_mariadb.yaml
```
getting this issue
```
Error from server (BadRequest): error when creating "examples/manifests/mariadb_v1alpha1_mariadb.yaml": MariaDB in version "v1alpha1" cannot be handled as a MariaDB: strict decoding error: unknown field "spec.imagePullPolicy"
```

**Comment by halkeye** [comment]
it looks like the examples have been updated to the new code in master, but release hasn't happened yet.

https://github.com/mariadb-operator/mariadb-operator/tree/helm-chart-0.24.0/examples/manifests seems to be the examples from the last release

**Comment by ashish-gupta-2** [comment]
I have tried with this version as well https://github.com/mariadb-operator/mariadb-operator/tree/helm-chart-0.24.0/examples/manifests

but still same issue 

@mmontes11  Any plan to release the master ?
So this issue can be fixed.

Or any other alternative option I can try?

**Comment by kasvith** [comment]
Im having the following issue with v0.0.24

```bash
admission webhook "vmariadb.kb.io" denied the request: json: cannot unmarshal object into Go struct field MariaDBSpec.spec.image of type string
```

**Comment by mmontes11** [comment]
Hey there @ashish-gupta-2! We have released [v0.0.25](https://github.com/mariadb-operator/mariadb-operator/releases/tag/v0.0.25) today. Let me know if you are still facing this problem.
